app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -71,6 +71,15 @@ def location_row(course_choice)
       {
         key: 'Location',
         value: "#{course_choice.offered_site.name}\n#{course_choice.offered_site.full_address}",
+        action: "location for #{course_choice.course.name_and_code}",
+        change_path: (if has_multiple_sites?(course_choice) && FeatureFlag.active?('edit_course_choices')
```

Comment:
This is a tiny bit more efficient:

```suggestion
        change_path: (if FeatureFlag.active?('edit_course_choices') && has_multiple_sites?(course_choice)
```